WordPress/Classes/Services/BlogService.m

```diff
@@ -20,14 +20,28 @@
 #import "NSString+XMLExtensions.h"
 #import "TodayExtensionService.h"
 
+NSString *const LastUsedBlogURLDefaultsKey = @"LastUsedBlogURLDefaultsKey";
+NSString *const BlogEntity = @"Blog";
+NSString *const EditPostViewControllerLastUsedBlogURLOldKey = @"EditPostViewControllerLastUsedBlogURL";
+NSString *const BlogName = @"blogName";
+NSString *const GetFeatures = @"wpcom.getFeatures";
+NSString *const VideopressEnabled = @"videopress_enabled";
+NSString *const XmlRpc = @"xmlrpc";
+NSString *const Standard = @"standard";
+NSString *const MinimumVersion = @"3.6";
+NSString *const DateCreatedGmt = @"date_created_gmt";
+NSString *const TimeZoneName = @"timezone";
```

Comment:
PS: on the downside we may need to change the names a bit... like `BlogTimeZoneNameKey`, `BlogGmtOffsetKey` and `BlogTimeZoneKey`.